### DISCUSSION TITLE
I cant write anything in osu on android

### DISCUSSION THREAD

**Discussion by QLeviathanQ** [ppy/osu#17373] [Q&A] [answered]
I cant write anything

**Comment by bdach** [comment, marked as answer]
see https://github.com/ppy/osu/discussions/15838, https://github.com/ppy/osu/discussions/15565, etc.

also at the least you could state which virtual keyboard you are using and on which device you are trying.

**Reply by QLeviathanQ** [reply]
I am using Gboard on a Redmi Note 10S device
